README.md

```diff
@@ -5,7 +5,7 @@ This repository contains externally maintained `.gitpod` files for projects on G
 
 The files are organized by repository name. If you need a different configuration for a fork the repository folder can contain a folder for the organisation.
 
-The configuration format is explained [here](https://docs.gitpod.io/40_Configuration.html#gitpod-file).
+The configuration format is explained [here](https://www.gitpod.io/docs/configuration/).

```

Comment:
Thanks! But actually, re-reading this sentence, I don't think that the configuration format is explained in https://www.gitpod.io/docs/configuration/ (that page seems to be more about the Setup Assistant / the general configuration process, as opposed to the specific `.gitpod.yml` format).

Maybe linking to https://www.gitpod.io/docs/config-gitpod-file/ would be more relevant / helpful here.

Reply:
Additionally, I notice that the paragraph above talks about a `.gitpod` file. I think this is outdated information, because this file is now called `.gitpod.yml`. 😄 Could you please fix it as well?

Reply:
Thanks! I have fixed it :slightly_smiling_face: